cdmsparkevents/eventloop.py

```diff
@@ -117,11 +122,19 @@ def _test_cts_connection(self):
     def _request_job(self, job_id: str) -> dict[str, Any]:
         return self._cts_request(f"admin/jobs/{job_id}")
 
-    def _cts_request(self, url_path: str) -> dict[str, Any]:
+    def _cts_request(
+            self,
+            url_path: str,
+            body: dict[str, Any] = None,
+            no_response_body: bool = False) -> dict[str, Any]:
```

Comment:
what's the purpose of `no_response_body` - looks like we just saved a `res.json()` call which is not very expensive.

Reply:
If there's no response body res.json() will fail I assume